### PR TITLE
Fix no carouselling when flipWait set to > 30 seconds

### PR DESCRIPTION
**Pull request by Jimadine**
See https://github.com/TabCarousel/TabCarousel/issues/66

### CODE REVIEW

File: src/javascripts/background.js
Comment (benjaminoakes) [inline src/javascripts/background.js:14]: Could we link to something future maintainers world find helpful?

File: src/javascripts/background.js
Comment (benjaminoakes) [inline src/javascripts/background.js:4]: ❤️

**Review by benjaminoakes — APPROVED** [review]
Thank you! This will go into the next release. 😀

**Comment by benjaminoakes** [comment]
Looks like this needs a tiny fix:

```
Run npm run lint

> lint
> eslint  ./src


/home/runner/work/TabCarousel/TabCarousel/src/javascripts/background.js
Error:   15:56  error  Missing semicolon  semi

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.

Error: Process completed with exit code 1.
```

File: src/javascripts/background.js
Comment (Jimadine) [reply src/javascripts/background.js:14]: We could, though I hope this will only be a temporary fix. https://stackoverflow.com/questions/66618136/persistent-service-worker-in-chrome-extension has good info but isn't very official. https://developer.chrome.com/docs/extensions/develop/concepts/service-workers might be a good official starting point.

Comment (benjaminoakes) [reply src/javascripts/background.js:14]: Just to step back:  I appreciate that you included a comment in your PR.  Very helpful!  :smile:  I'm coming from the perspective that future maintainers may not read all the context of our discussions on GitHub and may become confused or misunderstand.
 
Perhaps we should call out that this is temporary?   And really, I think anything you've mentioned works (including not changing the comment).

**Comment by benjaminoakes** [comment]
I'm not sure why the build step requires approval.  Should we make an issue for that?

**Comment by benjaminoakes** [comment]
:partying_face: 

>All checks have passed
>1 successful check

File: src/javascripts/background.js
Comment (Jimadine) [reply src/javascripts/background.js]: Thanks - I've tweaked the comment.

**Comment by Jimadine** [comment]
> I'm not sure why the build step requires approval. Should we make an issue for that?

Yeah, I'd say so - cheers!

File: src/javascripts/background.js
Comment (benjaminoakes) [inline src/javascripts/background.js:14]: Love it!

**Review by benjaminoakes — APPROVED** [review]

**Comment by benjaminoakes** [comment]
> > I'm not sure why the build step requires approval. Should we make an issue for that?
>
> Yeah, I'd say so - cheers!

Created https://github.com/TabCarousel/TabCarousel/issues/72

**Comment by benjaminoakes** [comment]
Thanks again!

**Pull request merged**
